### PR TITLE
Menu item sorting

### LINKED ISSUE #287: Nicer menu item ordering
Ordering menu items and putting them at the right place is quite difficult right now.
A better solution would be an ordering page where items can be ordered by dragging and dropping. This should include changing hierarchy levels.

### PR DESCRIPTION
**Pull request by janno42**
fix #287
allow drag and drop menu item sorting

### CODE REVIEW

File: _1327/main/templates/menu_items_list.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+				{% if item.document %}(<i>{% trans "Document" %}:</i> {{ item.document }}){% endif %}
+				{% if item.staff_only %}(<i>{% trans "Staff only" %}</i>){% endif %}
+				<button type="button" class="close" onclick="show_delete_modal({{ item.id }});"><span>&times;</span></button>
+			</div>
+			{% if item.subitems %}
+				{% include_menu_items_list item.subitems %}
+			{% endif %}
+		</li>
+	{% endfor %}
+</ol>
```
Comment (Nef10) [inline _1327/main/templates/menu_items_list.html]: new line at end

**Comment by Nef10** [comment]
I cannot move anything into an empty section

**Comment by janno42** [comment]
@Nef10 fixed moving last item, did some refactoring and made the unit test pass :)

**Comment by Nef10** [comment]
🎈

**Pull request merged**